EIPS/eip-7570.md

```diff
@@ -0,0 +1,48 @@
+---
+eip: 7570
```

Comment:
```suggestion
eip: to be assigned
```
Please note EIP/RIP/ERC numbers are manually assigned by editors & associates.  7570 has already been assigned.

Reply:
Hello,
Thank you for letting me know that it's my first time proposing an EIP.
I will make the necessary corrections.